app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+  def index
+
+  end
```

Comment:
to save 2 lines you can refactor this to def index; end

Reply:
I can do that in this PR - I just know this whole controller is going to be overridden in an upcoming PR when we build out the dashboard.
Thoughts?